doc/source/parameters/dem/post-processing.rst

```diff
@@ -2,7 +2,26 @@
 Post-processing
 ===============
 
-Lethe-DEM has built-in post-processing capabilities. All post-processing results are written as VTU file, generated with the same ``output frequency`` as the particles' results (see the :doc:`../dem/simulation_control` section of this guide). The name of the generated files are the same the one chosen for the particles' results plus the suffix ``-postprocess_data``. The files include:
+Lethe-DEM has built-in post-processing capabilities. All post-processing results are written as VTU, PTVU, and/or PVD files. 
+
+The post-processing subsection of the ``.prm`` file is according to the following example:
+
+.. code-block:: text
+
+ subsection post-processing
+  # Enable output of grid, granular temperature, and particles' average velocity
+  set Lagrangian post-processing = false
+  # Enable output of force chains
+  set force chains = false
+ end
+
+.. note::
+ By default, post-processing are set to ``false``.
+
+--------------------------
+Lagrangian post-processing
+--------------------------
+The ``Lagrangian post-processing`` enables VTU files output generated with the same ``output frequency`` as the particles' results (see the :doc:`../dem/simulation_control` section of this guide). The name of the generated files are the same the one chosen for the particles' results plus the suffix ``-postprocess_data``. The files include:
```

Comment:
```suggestion
The ``Lagrangian post-processing`` enables VTU files output generated with the same ``output frequency`` as the particles' results (see the :doc:`../dem/simulation_control` section of this guide). The name of the generated files are set by the ``output name`` parameter in the simulation control section plus the suffix ``-postprocess_data``. The files include:
```

Reply:
Got two extra comments/questions 

- It think the suffix for the post-processing lagrangian should be different since we have more than one post processing tool (force chain and this).

- The ``output name`` parameter is not documented in the simulation control page. (It could be fix in this PR since it is a tiny change, but we can fix it in an other one) (@blaisb should I open a issue for this?)

Reply:
Yeah you can open an issue and the first one with time can add a PR to add it to the documentation

Reply:
For the force chains, I'll change the name of the files for the output name plus the suffix "-force_chains". Do you want me to modify the names for post-processing lagrangian files in this PR?